Modules/Utils.cs

```diff
@@ -1589,11 +1589,10 @@ public static List<PlayerControl> GetPlayerListByIds(this IEnumerable<byte> Play
     {
         var PlayerList = PlayerIdList?.ToList().Select(x => GetPlayerById(x)).ToList();
 
-        return PlayerList.Any() ? PlayerList : null;
+        return PlayerList != null && PlayerList.Any() ? PlayerList : null;
```

Comment:
if getplayerbydids is used by a empty ienumerable<byte>, uses .Any() to determine it. (since linq methods would jsut return the empty one back and we would rather want to use null in this scenario so we know where the issue is if an error would occur)

if it is used by a null , any would throw a exception if it isn't checked for null.